src/test/java/uk/gov/justice/services/test/CantBeSerializedByJackson.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.services.test;
+

```

Comment:
should this be under 'uk.gov.justice.services.test.domain.event', as it represents an event right?

Reply:
Moved to util package as discussed